taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/controller/CamundaTaskEventsController.java

```diff
@@ -40,13 +76,106 @@ public Response getEvents(@QueryParam("type") final List<String> requestedEventT
     return Response.status(200).entity(camundaTaskEventListResource).build();
   }
 
-  @Path("/delete")
+  @Path(Mapping.URL_EVENT)
+  @GET
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getEvent(@PathParam("eventId") final int eventId) {
+
+    CamundaTaskEvent camundaTaskEvent = camundaTaskEventService.getEvent(eventId);
+
+    CamundaTaskEventResource camundaTaskEventResource =
+        camundaTaskEventResourceAssembler.toResource(camundaTaskEvent);
+
+    return Response.status(200).entity(camundaTaskEventResource).build();
+  }
+
+  @Path(Mapping.URL_DELETE_EVENTS)
   @POST
   @Consumes(MediaType.APPLICATION_JSON)
   public Response deleteEvents(String idsAsString) {
 
     camundaTaskEventService.deleteEvents(idsAsString);
```

Comment:
I'm not sure: did we agree to move the JSON handling to the controller? And if we did, to do it in this PR?

Reply:
We decided to disucc this in COP